crates/g3-config/src/lib.rs

```diff
@@ -70,10 +70,12 @@ pub struct AgentConfig {
     pub max_context_length: Option<u32>,
     pub fallback_default_max_tokens: usize,
     pub enable_streaming: bool,
+    pub allow_multiple_tool_calls: bool,
     pub timeout_seconds: u64,
     pub auto_compact: bool,
     pub max_retry_attempts: u32,
     pub autonomous_max_retry_attempts: u32,
+    pub verify_requirements_sha256: bool,
```

Comment:
could this be on by default (if it isn't already?) - as will never remember it otherwise, and I think it should just always do it right?